extensions/amp-video/1.0/storybook/Basic.js

```diff
@@ -0,0 +1,167 @@
+
+const {forwardRef, useRef, useImperativeHandle} = Preact;
+
+export default {
+  title: 'Video Wrapper',
+  component: VideoWrapper,
+  decorators: [withA11y, withKnobs],
+};
+
+const VideoTagPlayer = forwardRef((props, ref) => {
+  const {
+    children,
+
+    src,
+    muted,
+    loop,
+    controls,
+    poster,
+
+    'aria-label': ariaLabel,
+    album,
+    artist,
+    artwork,
+    title,
+
+    onLoad,
+    onLoadedMetadata,
+    onPause,
+    onPlaying,
+  } = props;
+
+  const videoNodeRef = useRef();
+
+  useImperativeHandle(ref, () => ({
+    play: () => videoNodeRef.current.play(),
+    pause: () => videoNodeRef.current.pause(),
+    getMetadata: () => ({
+      title: title || '',
+      artist: artist || '',
+      album: album || '',
+      artwork: [{src: artwork || poster || ''}],
+    }),
+  }));
+
+  return (
+    <video
+      ref={videoNodeRef}
+      aria-label={ariaLabel}
+      src={src}
+      muted={muted}
+      loop={loop}
+      controls={controls}
+      onPlaying={onPlaying}
+      onPause={onPause}
+      poster={poster}
+      onLoadedMetadata={onLoadedMetadata}
+      onCanPlay={onLoad}
+      style={{position: 'relative', width: '100%', height: '100%'}}
+    >
+      {children}
+    </video>
+  );
+});
+
+/**
+ * @return {PreactDef.Renderable}
+ */
+function ScrollSpacer() {
+  return (
+    <div
+      style={{
+        height: '100vh',
+        background: `linear-gradient(to bottom, #bbb, #bbb 10%, #fff 10%, #fff)`,
+        backgroundSize: '100% 10px',
+      }}
+    ></div>
+  );
+}
+
+export const _default = () => {
+  const componentGroup = 'Component';
+
+  const ariaLabel = text('aria-label', 'Video Player', componentGroup);
+  const autoplay = boolean('autoplay', true, componentGroup);
+  const controls = boolean('controls', true, componentGroup);
+  const mediasession = boolean('mediasession', true, componentGroup);
+  const noaudio = boolean('noaudio', false, componentGroup);
+  const loop = boolean('loop', false, componentGroup);
+  const poster = text(
+    'poster',
+    'https://amp.dev//static/inline-examples/images/kitten-playing.png',
+    componentGroup
+  );
+  const sources = object(
+    'sources',
+    [
+      {
+        src:
+          'https://amp.dev/static/inline-examples/videos/kitten-playing.webm',
+        type: 'video/webm',
+      },
+      {
+        src: 'https://amp.dev/static/inline-examples/videos/kitten-playing.mp4',
+        type: 'video/mp4',
+      },
+    ],
+    componentGroup
+  );
+
+  const metadataGroup = 'Metadata';
+
+  const artist = text('artist', '', metadataGroup);
+  const album = text('album', '', metadataGroup);
+  const artwork = text('artwork', '', metadataGroup);
+  const title = text('title', '', metadataGroup);
+
+  const containerGroup = 'Container';
+
+  const width = text('Width', '640px', containerGroup);
+  const height = text('Height', '360px', containerGroup);
+  const scrollSpacers = boolean('Scroll spacers', false, containerGroup);
```

Comment:
nit: Please change the description for this to be something more intuitive, as I did not know what to expect when I first checked the box 😅

Reply:
Any suggestions? :)

Reply:
This knob is gone now in favor of spacer config options. Let me know if the new set of options makes more sense.